docs/available_commands.md

```diff
@@ -14,10 +14,10 @@ Currently, the command adds or modifies the following pieces of information:
 
 * If the SBOM metadata doesn't specify an *author* from the SBOM, it will be set to `{"name": "automated"}`.
 * The *compositions* array will be overwritten with a new one which specifies a single *incomplete* aggregate. This aggregate contains all components, including the metadata component.
-* If a component does not have a *supplier*, the tool will try to infer the supplier from the fields (in order of precedence):
+* If a component does have a publisher and/or author but does not have a *supplier*, the tool will try to infer the `supplier.name` from the fields (in order of precedence):
```

Comment:
for consistency, I would either change the "a component" to "the component" or revert the change to "the" you did on [L#20](https://github.com/Festo-se/cyclonedx-editor-validator/pull/118/files#diff-bea07d8bee938360185b0b313b81496c7d1c302766e3d55def5f34279c161023R20)